src/Application/Regulation/View/RegulationOrderListItemView.php

```diff
@@ -10,6 +10,7 @@ public function __construct(
         public readonly string $uuid,
         public readonly string $identifier,
         public readonly string $status,
+        public readonly int $nbMoreLocations,
```

Comment:
(Détail) 'nb' c'est un peu français (nombre), en code anglais j'ai plus l'habitude de voir 'num' pour 'number'

```suggestion
        public readonly int $numMoreLocations,
```

Je trouverais plus naturel de ne passer qu'un `numLocations` total et de faire le traitement (si plus de 2, afficher ça - 1) dans le template uniquement, mais c'est subjectif